src/clockkit.cs

```diff
@@ -53,16 +53,20 @@ interface CLKComplicationDataSource {
 		[Export ("getTimelineEntriesForComplication:afterDate:limit:withHandler:")]
 		void GetTimelineEntriesAfterDate (CLKComplication complication, NSDate afterDate, nuint limit, Action<CLKComplicationTimelineEntry []> handler);
 	
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use WKRefreshBackgroundTask instead")]
```

Comment:
Please take a look at https://github.com/xamarin/xamarin-macios/wiki/BINDINGS#availability-attributes-messages as a refresher (:

Your message should be `Use 'WKRefreshBackgroundTask' instead.`